.vscode/settings.json

```diff
@@ -21,4 +21,5 @@
         "tools/pipelines/*.yml": "azure-pipelines"
     },
     "[azure-pipelines].customSchemaFile": "",
+    "debug.javascript.warnOnLongPrediction": false,
```

Comment:
Was this an intentional config change?  I don't know what it does, but if it was unintentional probably revert it.